programs/memo/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "mollusk-memo"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Can you align this manifest with the others, like the harness?
https://github.com/buffalojoec/mollusk/blob/6810892b26ac39feebe441c5f3d2daa8f6ba6735/harness/Cargo.toml#L3-L10